packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -59,10 +59,7 @@ export class ZipkinExporter implements SpanExporter {
     resultCallback: (result: ExportResult) => void
   ) {
     if (typeof this._serviceName !== 'string') {
-      this._serviceName = String(
-        spans[0].resource.attributes[SERVICE_RESOURCE.NAME] ||
-          this.DEFAULT_SERVICE_NAME
-      );
+      this._serviceName = this.DEFAULT_SERVICE_NAME;
```

Comment:
it doesn't make sense to set this value with every export, you can do it in constructor.